demos/gaze_estimation_demo/main.cpp

```diff
@@ -139,11 +143,12 @@ int main(int argc, char *argv[]) {
 
         HeadPoseEstimator headPoseEstimator(ie, FLAGS_m_hp, FLAGS_d_hp);
         LandmarksEstimator landmarksEstimator(ie, FLAGS_m_lm, FLAGS_d_lm);
+        EyeStateEstimator eyeStateEstimator(ie, FLAGS_m_es, FLAGS_d_es);
         GazeEstimator gazeEstimator(ie, FLAGS_m, FLAGS_d);
 
         // Put pointers to all estimators in an array so that they could be processed uniformly in a loop
-        BaseEstimator* estimators[] = {&headPoseEstimator, &landmarksEstimator, &gazeEstimator};
-
+        BaseEstimator* estimators[] = {&headPoseEstimator, &landmarksEstimator, &gazeEstimator, &eyeStateEstimator };
```

Comment:
I suggest disabling `gazeEstimator` if both eyes are classified as closed (so `eyeStateEstimator` should go before `gazeEstimator` in the array) and not drawing gaze result even for a single closed eye.